resources/lang/id/users.php

```diff
@@ -0,0 +1,321 @@
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'deleted' => '[pengguna yang dihapus]',
+
+    'beatmapset_activities' => [
+        'title' => 'Riwayat Modding :user',
+
+        'discussions' => [
+            'title_recent' => 'Diskusi terbaru yang baru dimulai',
+        ],
+
+        'events' => [
+            'title_recent' => 'Peristiwa terbaru',
+        ],
+
+        'posts' => [
+            'title_recent' => 'Postingan terbaru',
+        ],
+
+        'votes_received' => [
+            'title_most' => 'Paling banyak divote oleh (3 bulan terakhir)',
+        ],
+
+        'votes_made' => [
+            'title_most' => 'Paling banyak divote (3 bulan terakhir)',
+        ],
+    ],
+
+    'card' => [
+        'loading' => 'Memuat...',
+        'send_message' => 'kirim pesan',
+    ],
+
+    'login' => [
+        '_' => 'Masuk',
+        'locked_ip' => 'Alamat IP Anda dikunci. Mohon tunggu beberapa menit.',
+        'username' => 'Nama Pengguna',
+        'password' => 'Kata Sandi',
+        'button' => 'Masuk',
+        'button_posting' => 'Mencoba masuk...',
+        'remember' => 'Ingat perangkat ini',
+        'title' => 'Mohon masuk untuk melanjutkan',
+        'failed' => 'Gagal masuk',
+        'register' => 'Belum memiliki akun osu!? Buat sekarang',
+        'forgot' => 'Lupa kata sandi?',
+        'beta' => [
+            'main' => 'Akses beta saat ini dibatasi untuk pengguna istimewa.',
+            'small' => '(Supporter akan segera masuk)',
+        ],
+
+        'here' => 'di sini', // this is substituted in when generating a link above. change it to suit the language.
+    ],
+
+    'posts' => [
+        'title' => 'Postingan :username',
+    ],
+
+    'signup' => [
+        '_' => 'Daftar',
+    ],
+    'anonymous' => [
+        'login_link' => 'klik untuk masuk',
+        'login_text' => 'masuk',
+        'username' => 'Tamu',
+        'error' => 'Anda perlu masuk untuk melakukan ini.',
+    ],
+    'logout_confirm' => 'Apa Anda yakin akan keluar? :(',
+    'restricted_banner' => [
+        'title' => 'Akun Anda telah dibatasi!',
+        'message' => 'Selama dibatasi, Anda tidak dapat berinteraksi dengan pengguna lain dan skor hanya akan terlihat oleh Anda. Hal ini biasanya terproses secara otomatis dan akan diangkat dalam 24 jam. Jika Anda ingin mengajukan banding atas pembatasan Anda, mohon<a href="mailto:accounts@ppy.sh">hubungi layanan dukungan</a>.',
```

Comment:
Really minor stuff here but you should add a space between "mohon" and the "<a href..." (or else it'll look condensed on the web page)